plugins/GSdx/resource.h

```diff
@@ -132,6 +132,8 @@
 #define IDC_OSD_SIZE_TEXT               2191
 #define IDC_OSD_OPACITY_TEXT            2192
 // OpenGL Advanced Settings:
+#define IDC_OSD_MONPOS_TEXT             2193
+#define IDC_OSD_MONITOR_POS             2194
```

Comment:
I'd put these before the above comment.

Reply:
That was a fault with visual studio removing the comments and git diff. Fixed.